src/Controller/CRUDController.php

```diff
@@ -1094,9 +1099,9 @@ protected function getParameter($name)
      * @param int   $status
      * @param array $headers
      *
-     * @return Response with json encoded data
+     * @return JsonResponse with json encoded data
      */
-    protected function renderJson($data, $status = 200, $headers = [])
+    protected function renderJson($data, $status = 200, $headers = []): Response
```

Comment:
Before there wasn’t a return type, please remove it, otherwise it’s a bc break

Reply:
Oops. Reverted.